package.json

```diff
@@ -1,39 +1,42 @@
 {
-  "name": "apiary-console-seed",
-  "version": "1.0.0",
-  "description": "",
-  "main": "client/src/Seed/Seed.js",
-  "engines": {
-    "node": "7.2.0",
-    "npm": "4.0.2"
-  },
-  "scripts": {
-    "test": "concurrently -r -p none \"npm run test:sauce\" \"cd client && npm t\"",
-    "test:sauce": "wdio ./wdio.sauce.conf.js",
-    "test:local": "wdio",
-    "start": "concurrently -r -p none \"node index.js\" \"cd client && npm start\"",
-    "chromedriver": "chromedriver --port=4444 --url-base=/wd/hub",
-    "build:extension": "webpack ./extension/extension.js ./extension/build/extension.js"
+  "name": "@apiaryio/console-proxy",
+  "version": "0.0.0-semantically-released",
+  "main": "src/Seed/Seed",
+  "devDependencies": {
+    "classnames": "^2.2.5",
+    "condition-circle": "^1.5.0",
+    "jasmine-core": "^2.5.2",
+    "karma": "^1.5.0",
+    "karma-chrome-launcher": "^2.0.0",
+    "karma-firefox-launcher": "^1.0.1",
+    "karma-jasmine": "^1.1.0",
+    "karma-junit-reporter": "^1.2.0",
+    "karma-mocha-reporter": "^2.2.3",
+    "karma-sauce-launcher": "^1.1.0",
+    "karma-sourcemap-loader": "^0.3.7",
+    "karma-webpack": "^2.0.3",
+    "react-scripts": "^0.9.5",
+    "semantic-release": "^6.3.2"
   },
-  "author": "",
-  "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.17.1",
-    "cors": "^2.8.3",
-    "ejs": "^2.5.6",
-    "express": "^4.15.2",
-    "node-hamms": "git+https://github.com/apiaryio/node-hamms.git",
-    "xml": "^1.0.1"
-  },
-  "devDependencies": {
-    "chromedriver": "^2.28.0",
-    "concurrently": "^3.4.0",
+    "axios": "^0.16.0",
     "jschannel": "^1.0.2",
-    "wdio-jasmine-framework": "^0.3.0",
-    "wdio-junit-reporter": "^0.3.0",
-    "wdio-sauce-service": "^0.3.1",
-    "wdio-spec-reporter": "^0.1.0",
-    "webdriverio": "^4.6.2",
-    "webpack": "^2.3.3"
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "urijs": "^1.18.10"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "build:extension": "webpack ./extension/extension.js ./extension/build/extension.js",
+    "test:iframe": "karma start ./karma.iframe.conf.js",
+    "test:chrome": "karma start ./karma.chrome.conf.js",
+    "test": "npm run test:chrome && npm run test:iframe",
+    "eject": "react-scripts eject",
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+  },
+  "proxy": "http://localhost:3001/",
```

Comment:
It's [a cra specific part](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#proxying-api-requests-in-development)